python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -453,6 +461,13 @@ def determinePsf(
             stars.append(piff.Star(data, None))
 
         if self.config.piffPsfConfigYaml is None:
+            # The following is mostly accommodating unittests that don't have
```

Comment:
In a later ticket, we should just set a placeholder filter objects instead of having to special case them here.